docroot/sites/default/settings.php

```diff
@@ -1,19 +1,22 @@
 <?php
 
-// @codingStandardsIgnoreFile
-
-/**
- * Include our ACSF settings if we are on an ACSF environment.
- */
-if (!empty($_ENV['AH_SITE_ENVIRONMENT']) &&
-  function_exists('gardens_site_data_get_filepath')) {
-  // ===== Added by acsf-init, please do not delete. Section start. =====
-$_acsf_infrastructure = include dirname(__FILE__) . '/acsf.settings.php';
-if ($_acsf_infrastructure === 'acsf-infrastructure') {
+if (isset($_ENV['AH_ENVIRONMENT_TYPE']) && $_ENV['AH_ENVIRONMENT_TYPE'] === 'meo') {
+  $settingsFiles = [
+    DRUPAL_ROOT . '/sites/settings/pre_includes.php',
+    DRUPAL_ROOT . '/sites/settings/post_includes.php',
+    DRUPAL_ROOT . '/sites/settings/post_memcache.php',
+    DRUPAL_ROOT . '/sites/settings/post_new_relic.php'
+  ];
+  foreach ($settingsFiles as $settingsFile) {
+    if (file_exists($settingsFile)) {
+      // phpcs:ignore
+      require $settingsFile;
+    }
+  }
   return;
 }
```

Comment:
2 things:
1. The folder location means they are getting loaded in [global.settings.php](https://github.com/NCIOCPL/cgov-digital-platform/blob/1c7a3e3b34c284e5746f43081de5a3f03df7c5df/docroot/sites/settings/global.settings.php)
   * These probably come first
   * We need to plot the order again of all the includes. 
2. Return? Really? So are we skipping our config settings?